examples/ping-pong/src/main.rs

```diff
@@ -122,11 +131,17 @@ async fn main() {
 
     GOSSIP_AGENT
         .get()
-        .unwrap()
-        .register_handler(Arc::new(Mutex::new(radio_handler)));
+        .expect("Could not retrieve gossip agent")
+        .register_handler(Arc::new(Mutex::new(radio_handler)))
+        .expect("Could not register handler");
 
     loop {
-        let block_number = U64::as_u64(&provider.get_block_number().await.unwrap());
+        let block_number = U64::as_u64(
+            &provider
+                .get_block_number()
+                .await
+                .expect("Could not get block number from provider"),
```

Comment:
I don't think a retry is necessary since the wait time of the loop is 5 secs, so we'll check for each block twice, which is enough of a retry mechanism imo

Reply:
should be good now ✅